verifier/src/x509/chain.rs

```diff
@@ -333,4 +342,45 @@ mod pkits {
 
         assert_eq!(signing_key, expected_key);
     }
+
+    #[test]
+    fn invalid_ca_signature_test_4_1_2() {
```

Comment:
After finishing this section of PKITS, thikning maybe the tests go to integration tests and name each test module for the PKITS pdf section i.e. `pkits-4_1.rs` or similar.

Reply:
Done in #75